README.md

```diff
@@ -211,7 +211,7 @@ Update `enableMistral` to `true` in [cdk.json](./cdk/cdk.json), and run `cdk dep
 > [!Important]
 > This project focus on Anthropic Claude models, the Mistral models are limited supported. For example, prompt examples are based on Claude models. This is a Mistral-only option, once you toggled to enable Mistral models, you can only use Mistral models for all the chat features, NOT both Claude and Mistral models.
 
-### Configure text generation
+### Configure default text generation
```

Comment:
From a user's perspective, `config.py` should no longer handle custom settings. Could you please update the documentation to change to modifying the frontend TypeScript instead?

Reply:
The generation parameters are feffered from config.py if no bot specified, so please ignore above comment 🙇